pkg/controllerhelpers/scylladbmanager.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2025 ScyllaDB
+
+package controllerhelpers
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/scylladb/scylla-manager/v3/pkg/managerclient"
+	scyllav1alpha1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1alpha1"
+	"github.com/scylladb/scylla-operator/pkg/naming"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+func GetScyllaDBManagerClient(_ context.Context, _ *scyllav1alpha1.ScyllaDBManagerClusterRegistration) (*managerclient.Client, error) {
+	url := fmt.Sprintf("http://%s.%s.svc/api/v1", naming.ScyllaManagerServiceName, naming.ScyllaManagerNamespace)
+	managerClient, err := managerclient.NewClient(url, func(httpClient *http.Client) {
+		httpClient.Transport = http.DefaultTransport
```

Comment:
Using global `http.DefaultTransport` might be error-prone. I suggest cloning it. If we want to share the transport between all manager clients (i.e., to cache connections), we might create our own global in this package and use it solely for that purpose.
```suggestion
		httpClient.Transport = http.DefaultTransport.(*http.Transport).Clone()
```

Reply:
This code is only moved, not added in this PR. It comes directly from the existing implementation (standalone manager-controller). I suggest we handle it separately, similar to max sync duration etc. We could also reevaluate why we're not using the default manager client config.